app/javascript/components/ansible-repository-form/helpers.jsx

```diff
@@ -0,0 +1,17 @@
+
+// NOTE: ddfUrlRegex is the regex output of https://github.com/data-driven-forms/react-forms/blob/3199d495361313616901f52ba2da6a21cb3d3a50/packages/react-form-renderer/src/validators/url-validator.js#L60-L80 this file 
+const ddfUrlRegex = /^((([-a-zA-Z0-9%._+~#=])@)(([-a-zA-Z0-9%._+~#=])(?:(?:).)*))|((?:\S+(?::\S*)?@)?)(?:((?:(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0)\.){3}(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0))|(((?:[0-9A-Fa-f]{1,4}))((?::[0-9A-Fa-f]{1,4}))*::((?:[0-9A-Fa-f]{1,4}))((?::[0-9A-Fa-f]{1,4}))*|((?:[0-9A-Fa-f]{1,4}))((?::[0-9A-Fa-f]{1,4})){7}|::1|1::)|((?![-_])(?:[-\w\u00a1-\uffff]{0,63}[^-_]\.)+(?:[a-z\u00a1-\uffff]{2,}\.?))|(localhost))((?::\d{2,5})?)((?:[/][^\s?#]*)?)((?:[?][^\s#]*)?)((?:[#]\S*)?)$/i;
+// sshReged is essentially the same as ddfUrlRegex but we've removed the protrocal and modified PATH to have a colon instead of a slash
```

Comment:
```suggestion
// sshRegex is essentially the same as ddfUrlRegex but we've removed the protocol and modified PATH to have a colon instead of a slash
```